translations.properties

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (C) 2003-2018 eXo Platform SAS.
+#
+# This is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as
+# published by the Free Software Foundation; either version 3 of
+# the License, or (at your option) any later version.
+#
+# This software is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this software; if not, write to the Free
+# Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+# 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+#
+
+# Mandatory property that indicate the base directory where each file is located in Crowdin (must end with /).
+baseDir=add-ons/onlyoffice/
+
+# Files
+Onlyoffice.properties=services/src/main/resources/locale/onlyoffice/Onlyoffice_en.xml
```

Comment:
Did you check if there are useless labels in the files ?

Reply:
No, i will check now

Reply:
It's ok for Onlyoffice_en.xml file